spec/views/cases/_case_request_html_slim_spec.rb

```diff
@@ -36,20 +36,8 @@
   let(:empty_message) { "" }
 
   describe 'Displaying help message when the request of an offender-sar case is empty' do
-    let(:offender_sar_case) {
-      (create :offender_sar_case, subject_aliases: 'John Smith',
-              date_of_birth: '2019-09-01', message: empty_message).decorate
-    }
+    let(:offender_sar_case) { (create :offender_sar_case, message: empty_message).decorate }
```

Comment:
Cool - we don't need to decorate the case, either. 

If you're using a let to create context, it should be inside the describe block to which it belongs. In this case I'd lose the let and just put the empty string in the create statement but if you _really_ want the additional context from the variable name, perhaps you could move the `let(:empty_message) { "" }` to just above this line

```suggestion
    let(:offender_sar_case) { create :offender_sar_case, message: '' }
```